README.md

```diff
@@ -94,12 +94,13 @@ Notable features:
 - Support for x86-64 and ARM runners with unlimited concurrency.
 - **Live debug failed jobs over SSH.**[^warp-debug]
 - Discounts for open-source projects and young startups, and referral programs to get extra minutes.[^warp-pricing]
-- Minimal documentation and unclear hardware specs. ⚠️
-- **(Coming soon)** Windows, macOS, and custom base images support (~Jan 2024). 🚧
+- Support for macOS runners on M2 Pros.[^warp-runners]
+- **(Coming soon)** Automated container layer caching and AI powered analytics (~Q1 2024). 🚧
```

Comment:
Is Windows and custom base image support still planned? I'd prefer to showcase that instead of AI analytics, as that kind of feature is hard to compare against competitors unless already available.

Reply:
We've deprioritized it for now because of lack of demand. We may take them up again in the future.

Reply:
Sounds good!